### PR TITLE
Add engine 19.03.13 release notes

### PR DESCRIPTION
**Pull request by chris-crone**
@thaJeztah @tiborvass PTAL

### CODE REVIEW

**Review by thaJeztah — APPROVED** [review]
LGTM, thanks!

File: engine/release-notes/index.md [hunk 1]
```diff
@@ -22,14 +22,15 @@ for Docker Engine.
 
 # Version 19.03
 
-## 19.03.13 (2020-09-16)
+## 19.03.13
+2020-09-16
```
Comment (thaJeztah) [inline engine/release-notes/index.md]: Ah, thanks @usha-mandya, overlooked that

**Review by thaJeztah — APPROVED** [review]
still LGTM

**Review by usha-mandya — APPROVED** [review]

File: engine/release-notes/index.md (hunk 1 above)
Comment (usha-mandya) [reply engine/release-notes/index.md]: No worries :)

**Pull request merged**